rust/connlib/tunnel/proptest-regressions/tests.txt

```diff
@@ -35,4 +35,6 @@ cc 86428dfe2e071f0160a99f240534b0db98f1d4de1a1829f1e9661bcb45c240d7
 cc b5a13ab3fa2a875bc9da0f5532064829d02bca6d90a2d96e728171dec3886812 # shrinks to (initial_state, transitions, seen_counter) = (ReferenceState { now: Instant { tv_sec: 18331, tv_nsec: 298225413 }, utc_now: 2024-06-05T02:49:28.195874121Z, client: SimNode { id: ClientId(ac8ae3a8-8006-0a95-a477-2828c8c96872), state: PrivateKey("54daca72a082bac6ff820bd7abf06e3c813b00164a19e1f111abca6b4fe5ca8c"), ip4_socket: Some(146.129.166.224:7547), ip6_socket: None, tunnel_ip4: 100.64.207.103, tunnel_ip6: fd00:2021:1111::d:3454 }, gateway: SimNode { id: GatewayId(ba5047ca-9daa-e02c-7299-d60f8a8980fa), state: PrivateKey("f0a3c066bd990fd90306d939048fcb96a9f2a372caf201044fde1689f64165dc"), ip4_socket: Some(60.149.176.66:49811), ip6_socket: Some([::ffff:0.0.0.0]:64528), tunnel_ip4: 100.70.217.183, tunnel_ip6: fd00:2021:1111::1c:c9fb }, relay: SimRelay { id: RelayId(42ef0f63-fb7b-edc4-6eec-51d88af32c9b), ip_stack: Dual { ip4: 197.243.220.18, ip6: ::ffff:213.206.92.5 }, allocations: {} }, system_dns_resolvers: [::ffff:251.83.154.133, 216.113.154.11], upstream_dns_resolvers: [], client_cidr_resources: {}, client_dns_resources: {}, client_dns_records: {}, client_connected_cidr_resources: {}, global_dns_records: {}, expected_icmp_handshakes: [], expected_dns_handshakes: [] }, [AddDnsResource { resource: ResourceDescriptionDns { id: ResourceId(00000000-0000-0000-0000-000000000000), address: "aaa.aaa", name: "aaaa", address_description: "abbz", sites: [Site { name: "aaaa", id: SiteId(52c3579a-f101-58ed-2a60-4c8801269612) }] }, resolved_ips: {91.158.158.211, 247.85.48.207, a58b:3256:76fe:e116:b62b:9fd9:1b0f:a8d3} }, SendDnsQuery { r_idx: Index(11437376390215257704), r_type: AAAA, query_id: 35520, dns_server_idx: Index(15689498379236676730) }, SendDnsQuery { r_idx: Index(12180571831563445147), r_type: A, query_id: 25178, dns_server_idx: Index(2710210489327420884) }, SendICMPPacketToResource { idx: Index(6687070428275117221), seq: 42506, identifier: 47841, src: TunnelIp6 }], None)
+cc ad82010f5fe572646ffded6d071392a4e67ebcdd7c2add2cd26d6433ad5a16e4
+cc bebf2584fa9a47eafd09973073038358ade7f06d46200b5c4beaee1ff2a05988
```

Comment:
If the proptesting code changes, these might not replicate an issue, right? Like how when Minecraft changes their worldgen code, it's not compatible with old save files or something.

Reply:
I think if the _strategies_ change, then it might not replicate the issue. Ultimately, it is not that important I think. We are running enough cases constantly that bugs will surface eventually.